File: app/src/main/java/net/imatruck/dsmanager/utils/PercentFormatter.java
package net.imatruck.dsmanager.utils;


public class PercentFormatter {

    /**
     * Returns the percentage of {@code a} over {@code b}
     * @param a current value
     * @param b total value
     * @return percentage
     */
    public static double toPercent(double a, double b) {
        return Math.round((a / b) * 10000) / 100f;
    }
}

File: app/src/main/java/net/imatruck/dsmanager/utils/PercentFormatter.kt
package net.imatruck.dsmanager.utils


object PercentFormatter {

    /**
     * Returns the percentage of `a` over `b`

     * @param a current value
     * *
     * @param b total value
     * *
     * @return percentage
     */
    fun toPercent(a: Double, b: Double): Double {
        return (Math.round(a / b * 10000) / 100f).toDouble()
    }
}
